frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/radiogram-list/radiogram-list.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="d-flex flex-column w-100 h-100">
+    <div class="flex-shrink-1 p-4 pb-0">
+        <h5>Funksprüche</h5>
+        <div class="mb-3 form-check form-switch">
+            <label class="from-check-label" for="showDoneSwitch">
+                Durchgesagte Funksprüche anzeigen
+            </label>
+            <input
+                [ngModel]="showDone$ | async"
+                (ngModelChange)="changeShowDone($event)"
+                type="checkbox"
+                role="switch"
+                class="form-check-input"
+                id="showDoneSwitch"
+            />
+        </div>
+    </div>
+    <div class="overflow-auto p-4">
+        <ng-container
+            *ngIf="(visibleRadiograms$ | async)?.length; else noRadiograms"
+        ></ng-container>
+        <app-radiogram-card
+            *ngFor="
+                let radiogram of visibleRadiograms$ | async;
+                trackBy: 'id'
+                    | appTrackByProperty : (visibleRadiograms$ | async)
+            "
+            [radiogramId]="radiogram.id"
+        ></app-radiogram-card>
+        <ng-template #noRadiograms>
+            <div
+                *ngIf="showDone$ | async; else noNotDoneRadiograms"
+                class="text-muted mt-3 mb-3"
+            >
+                Es sind noch keine Funksprüche vorhanden.
+            </div>
+            <ng-template #noNotDoneRadiograms>
+                <div class="text-muted mt-3 mb-3">
+                    Es sind keine ausstehenden Funksprüche vorhanden.
+                </div>
+            </ng-template>
+        </ng-template>
```

Comment:
Those two have a different width, causing the border between the sidebar and the simulated region to wiggle when changing the switch

Reply:
I do not think we want this to happen, so please give the sidebar a fixed portion of the modal

Reply:
Done